src/extension.ts

```diff
@@ -316,13 +316,13 @@ export class Extension implements RunHooks {
       await this._models.createModel(workspaceFolderPath, uriToPath(configFileUri), playwrightInfo);
     }
 
-    await this._models.ensureHasEnabledModels();
+    this._models.ensureHasEnabledModels();
```

Comment:
The methods inside were sync - hence changed the method signature to sync as well.